docs/Developers/Newcomers_Guide.rst

```diff
@@ -298,7 +253,56 @@ Ask a developer to check if it should be reclassified as `low` difficulty.
       please check the following link:
       `PEP8 Style Guide for Python code <https://www.python.org/dev/peps/pep-0008/>`_
 
-Step 3. Creating a Fork and Testing Your Changes
+Step 3. Verifying the Issue Exists
+----------------------------------
+
+You must ensure that the issue you would like to work on exists. You should
+look through the code or documentations to spot the region where the issue
+resides.
+This is an important step while taking up an issue which is often overlooked.
+
+.. important::
+
+    * If the issue is a bug in the source code then that shall be confirmed by
+      executing the relevant code or files. You should also figure out how will
+      you rectify this bug.
+
+    * If the issue is about some discrepancy in the documentation, then locate
+      the region in the documentation where it exists. Find out how it can be
+      removed.
+
+    * If the issue requires some additional functionality to be added, then
+      check whether it is possible and does not conflict with the existing
+      ones.
+
+    * If the issue requires the code to have some comments, then provide them
+      in plain and easy-to-understand language.
+
+    * If the issue is some grammatical or spelling error either in comments or
+      documentation, then figure out how it can be corrected. Wrong grammar or
+      a misspelt word can make things difficult to interpret.
+
+You must verify that the issue actually exists and work on it only if you know
+how to fix it. While verifying the issue, you must start thinking in the
+direction of resolving it as well. If you are able to find a find fix for the
+issue, then you can definitely claim it!
+
+The following should also be done to confirm the validity of the issue:
+
+- Check for 'closed' tag on the issue. If you can see the red-coloured 'closed'
+  tag, then it means that the issue is closed. An issue is closed when a pull
+  request resolving that issue is merged into the repository's default branch.
+- Check if pull request exists or not. If it exists, then you must look for
```

Comment:
`pull request` -> `an open pull request`.

Reply:
Even if there is an open pull request, sometimes people can still claim the issue. e.g. the open pull request has been stale for months. Then it is very likely the previous developer either forgets it or gives it up. In this case, you need to claim that you would like to do the issue and also ask for help on our gitter channel.

Reply:
So what should I write here?